app/models/homePage/HomePageViewModel.scala

```diff
@@ -63,4 +64,6 @@ case class ReturnsTileViewModel(currentTaxYear: TaxYear, iTSASubmissionIntegrati
 case class YourBusinessesTileViewModel(displayCeaseAnIncome: Boolean, incomeSourcesEnabled: Boolean,
                                        incomeSourcesNewJourneyEnabled: Boolean)
 
-case class AccountSettingsTileViewModel(currentTaxYear: TaxYear, reportingFrequencyEnabled: Boolean, currentYearITSAStatus: ITSAStatus)
\ No newline at end of file
+case class AccountSettingsTileViewModel(currentTaxYear: TaxYear, reportingFrequencyEnabled: Boolean, currentYearITSAStatus: ITSAStatus)
+
+case class PenaltiesAndAppealsTileViewModel(penaltiesAndAppealsEnabled: Boolean)
```

Comment:
Please add "Penalty points" to this view model, should be fine to create a fake service to return fixed value while integration task still in progress;

Reply:
also I don't think we need penaltiesAndAppealsEnabled field in the first place, we can make penalties points optional and and create different model depending on the FS status etc